Dockerfile

```diff
@@ -1,9 +1,8 @@
-FROM golang:1.5
-
+FROM golang:1.8
 RUN mkdir /etc/incus
 
-ADD . /go/src/github.com/Imgur/incus
-WORKDIR /go/src/github.com/Imgur/incus
+ADD . /go/src/github.com/jtaylor32/incus
+WORKDIR /go/src/github.com/jtaylor32/incus
```

Comment:
Can you be sure to change this back? When working on a fork with Golang I find this handy: https://stackoverflow.com/questions/14323872/using-forked-package-import-in-go